app/internal/event.py

```diff
@@ -9,46 +9,56 @@
 
 from app.database.models import Event
 
-ZOOM_REGEX = re.compile(r'https://.*?\.zoom.us/[a-z]/.[^.,\b\s]+')
+ZOOM_REGEX = re.compile(r"https://.*?\.zoom.us/[a-z]/.[^.,\b\s]+")
 
 
 def raise_if_zoom_link_invalid(vc_link):
     if ZOOM_REGEX.search(vc_link) is None:
-        raise HTTPException(status_code=HTTP_400_BAD_REQUEST,
-                            detail="VC type with no valid zoom link")
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST,
+            detail="VC type with no valid zoom link",
+        )
 
 
 def get_invited_emails(invited_from_form: str) -> List[str]:
     invited_emails = []
-    for invited_email in invited_from_form.split(','):
-        invited_email = invited_email.strip()
-        try:
-            validate_email(invited_email, check_deliverability=False)
-        except EmailSyntaxError:
-            logging.exception(f'{invited_email} is not a valid email address')
-            continue
-        invited_emails.append(invited_email)
+    if not invited_from_form:
+        invited_emails.append("")
```

Comment:
`return [""]` instead, and unindent the `else` clause

Reply:
Done :) (I used try, except else inside the for loop)